studio/components/interfaces/Database/Triggers/CreateTrigger.tsx

```diff
@@ -373,33 +373,13 @@ const CreateTrigger: FC<CreateTriggerProps> = ({ trigger, visible, setVisible })
             </CreateTriggerContext.Provider>
           </div>
         ) : (
-          <NoTableState />
+          <NoTableState message="You will need to create a table first before you can make a trigger" />
```

Comment:
any reason why we shouldn't use ProductEmptyState directly, and instead creating a NoTableState as a wrapper around that? just feels unnecessary

Reply:
hi, sorry for the late reply. Previously, the CreateTrigger component was doing NoTableState, and from this issue, the  auth/policies.tsx page needed to do the same (but with a different message). So I decided to make a separate component for NoTableState to be used by CreateTrigger and the auth/policies.tsx page